instrumentation/pekko-http-1.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+  id("otel.scala-conventions")
+}
+
+muzzle {
+  pass {
+    group.set("org.apache.pekko")
+    module.set("pekko-http_2.12")
+    versions.set("[1.0, 2.0)")
```

Comment:
the only version of `pekko-http` seems to be `1.0.0`, I'd use `versions.set("[1.0,)")`

Reply:
Good catch - that's fixed now.